src/components/Map/Map.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+  act,
+  fireEvent,
+  render as renderRtl,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { mockMediaQuery } from '@test/testUtils';
+
+import type { MapProps, Location } from './Map';
+import { Map } from './Map';
+
+const mediaQueryBreakPoint = 500;
+const { setScreenWidth } = mockMediaQuery(mediaQueryBreakPoint);
```

Comment:
These can also be removed, theres no changes when rendering in mobile view here 🙂